data/json/items/comestibles/med.json

```diff
@@ -23,7 +23,7 @@
     "id": "adderall",
     "type": "COMESTIBLE",
     "comestible_type": "MED",
-    "name": { "str_sp": "Adderall" },
+    "name": { "str_sp": "dextro-/amphetamine" },
```

Comment:
```suggestion
    "name": { "str_sp": "dextroamphetamine" },
```

Reply:
but Adderall is the combination of both, dextroamphetamine and Amphetamine. 50/50 as far as I know, just using dextraamphetamine would cut the other half away.